modules/jenkins/manifests/init.pp

```diff
@@ -9,40 +9,24 @@
 class jenkins (
   $repo_url = 'https://pkg.jenkins.io/redhat-stable',
   $key_url  = 'https://pkg.jenkins.io/redhat-stable/jenkins.io.key',
-  $dports = ['8080', '9000'],
+  $dports   = ['8080', '9000'],
+  $plugins  = ['sonar', 'maven-plugin', 'jquery']
```

Comment:
it should be empty by default.
All other plugins should  be declared as defined type calls on PROFILE level.